metadata-service/auth-impl/src/test/java/com/datahub/authorization/PolicyEngineTest.java

```diff
@@ -1,6 +1,7 @@
 package com.datahub.authorization;
 
 import com.datahub.authentication.Authentication;
+import com.datastax.oss.driver.shaded.guava.common.collect.ImmutableList;
```

Comment:
This is definitely the wrong import

Reply:
My imports on OSS are not working. Thanks for the catch